source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -67,6 +67,8 @@ type Props = {
   onOpenDialogAction: Function,
   onUnsetActiveAssetFingerprint: Function,
   onExternalLinkClick: Function,
+  addressFromSameWallet: boolean,
```

Comment:
@aleksandardjordjeviciohk please rename to `isAddressFromSameWallet`

Reply:
Fixed